tests/snapshot-gating.fmf

```diff
@@ -10,10 +10,15 @@
 
 summary: LLVM Tests for snapshot gating
 prepare:
-  - how: install
-    copr: "@fedora-llvm-team/$COPR_PROJECT $COPR_CHROOT"
-    # Lower the priority of the testing-farm-tag-repository so that our copr repo is picked up.
-    # See: https://docs.testing-farm.io/Testing%20Farm/0.1/test-environment.html#_tag_repository
+  # dnf5's copr plugin has trouble resolving the runtime dependency repos, so
+  # we need to use dnf4. Check https://github.com/fedora-copr/copr/issues/3387
+  - name: Enable copr repo
+    how: shell
+    script: |
+      dnf install -y dnf-plugins-core
+      dnf-3 -y copr enable @fedora-llvm-team/$COPR_PROJECT $COPR_CHROOT
```

Comment:
You said in the initial comment that you wanted to use dnf4. But here you use dnf3

Reply:
True, the naming is pretty confusing. The "3" in "dnf-3" refers to the python-3 implementation of dnf4. So, `dnf-3` command is in fact dnf4:
```
# rpm -qf $(which dnf-3)
python3-dnf-4.21.1-1.fc42.noarch
```

Reply:
Yes that makes no sense whatsoever. :)

Reply:
Anything else to review here, or any changes requested?